brainiak/matnorm_models/noise_cov_gp.py

```diff
@@ -0,0 +1,53 @@
+from .noise_covs import NoiseCovBase
```

Comment:
I've separated it intentionally since it's an optional shim for using GPflow kernels and not another covariance. Can you provide more justification for why we should combine them? (I do agree about combining the kronecker covariance with the rest).